src/radix_connect/mobile/relay_service/models/response/wallet_interaction_requests.rs

```diff
@@ -0,0 +1,57 @@
+use crate::prelude::*;
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+struct WalletInteractionRequests(Vec<BagOfBytes>);
```

Comment:
Os the plural correct? that this is many requests?

If we plan on UniFFI exporting `WalletInteractionRequests` then we might wanna use the `secret_magic` dance here, otherwise `WalletInteractionRequests` will become `public typealias WalletInteractionRequests = Data` in Swift, and we can accidantally use `Data()` in Swift, which is wrong. 

If we dont plan on UniFFI exporting it, we ought to use `Vec<u8>` as per earlier comment.

Reply:
Yes, the RadixConnectRelay returns a list of requests for a given session id, no only one.

And no, this is not meant to be exposed through UniFFI.